generators/app/index.js

```diff
@@ -130,6 +130,7 @@ module.exports = class extends BaseGenerator {
             defaults: false
         });
 
+        this.availableGeneratorConfig = this.options.availableGeneratorConfig;
```

Comment:
@MathieuAA I think this might be duplication of the `configOptions` param that is passed around, can you check?
The purpose of `configOptions` was to pass context around when composing so please do see if you are able to use it without introducing a new param

Reply:
roger that, thanks for the tip